lib/resque/scheduler_patch.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'scheduler/delaying_extensions_old'
+module Resque
+  module SchedulerPatch
```

Comment:
Is this module optional? Would be good to enable it for a single pod (pod1 maybe) in Shopify for a few days before we commit to it everywhere for multiple days

Reply:
We cant enable it per pod sadly. However my plan is to enable it in cloud2 with only pod 21 there.